.vscode/tasks.json

```diff
@@ -0,0 +1,18 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
```

Comment:
Is it just me or these three line's background are different?

Also, I think we shouldn't put specific code editor's setting in the codebase. Is there any reason for this?

Reply:
Missed it. Will ad to the .gitignore